dsp/window/window.go

```diff
@@ -330,6 +330,54 @@ func (g Gaussian) Transform(seq []float64) []float64 {
 	return seq
 }
 
+// The values for specrtal leakage come from
```

Comment:
Typo.

Reply:
Yes, and please move the text below the table so users can see it as I have [here](https://github.com/gonum/gonum/pull/1437#discussion_r484182529). They may not be able to get the document, but they should see where it comes from.

Reply:
I had placed it outside the docstring intentionally but it's not a hill I want to die on.  Updated.